packages/head/src/head.tsx

```diff
@@ -9,6 +9,7 @@ export const Head = React.forwardRef<HeadElement, Readonly<HeadProps>>(
   ({ children, ...props }, forwardedRef) => (
     <head ref={forwardedRef} {...props}>
       <meta httpEquiv="Content-Type" content="text/html charset=UTF-8" />
+      <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
We shouldn't force adding Tailwindcss. Not everyone uses Tailwindcss

Reply:
Maybe it's better to add support for Tailwind in a different PR than the addition to Adonis instruction in the docs since they are not related and in different packages.

Reply:
Resolved